src/library_webgl.js

```diff
@@ -8,9 +8,23 @@ var LibraryGL = {
 
   // For functions such as glDrawBuffers, glInvalidateFramebuffer and glInvalidateSubFramebuffer that need to pass a short array to the WebGL API,
   // create a set of short fixed-length arrays to avoid having to generate any garbage when calling those functions.
-  _tempFixedLengthArray__postset: 'for (var i = 0; i < 32; i++) __tempFixedLengthArray.push(new Array(i));',
+  _tempFixedLengthArray__postset: 'for (var ____n = 0; ____n < 32; ++____n) __tempFixedLengthArray.push(new Array(____n));',
```

Comment:
why this change?

Reply:
The `var` inside the `for` init block applies to the outer (global) scope. Closure does not let the following function compile:
```js
for(var i = 0; i < 10; ++i);
for(var i = 0; i < 10; ++i);
```
but it complains `i` is defined twice.

Reply:
Hmm that is unfortunate. But is `___n` better? seems like we'll need a new id in each place we have a loop. And multiple underscores specifically worry me as maybe a typo with the number of underscores could happen. How about just `j`?

A singleton `var i;` on top and then loops not having a `var` at all would be more compact, and avoid this. But it's also an error-prone pattern.

Reply:
Reverted back to `i` and applied a Closure suppression instead.